docs/DeveloperGuide.md

```diff
@@ -374,11 +371,79 @@ otherwise)
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. ClickConnect shows an error message.
 
       Use case resumes at step 2.
 
-*{More to be added}*
+**System**: ClickConnect
+
+**Use case**: UC02 - See usage instructions
+
+**Actor**: User
+
+**MSS**
+
+1. User opens ClickConnect application
+2. User selects option to view usage instructions from main menu
+3. ClickConnect displays the usage instructions such as how to add, edit, delete or search for contacts
+4. User reads the instructions to understand how to use the ClickConnect
+5. Use case ends
+
+**Extensions**
+
+* 2a. The application does not have a direct link to usage instructions visible:
+  
+    * 2a1. ClickConnect guides the user to the help section where instructions can be found
+    * Use case resumes
+
+**System**: ClickConnect
+
+**Use case**: UC03 - Add new contact
+
+**Actor**: User
+
+**MSS**
+
+1. User opens ClickConnect application
+2. User requests to add a new contact
+3. ClickConnect displays a success message after the new contact is successfully added
+4. Use case ends
+
+**Extensions**
+
+* 2a. User does not know what to type to add a contact
```

Comment:
Would this involve any interaction between the user and the system? If the user is unsure of how to add a contact in step 2a, would the user know how carry out step 2 - "User requests to add a new contact"?